source/regions/flags.rst

```diff
@@ -368,3 +377,6 @@ Miscellaneous
     send-chat,state,Whether players can send chat
     receive-chat,state,Whether players can receive chat
     potion-splash,state,Whether potions can have splash effects
+
+.. note::
+    WorldGuard can only limit chat or commands on the server WorldGuard is running on. If you are using some sort of proxy that handles commands or the chat, WorldGuard won't be able to affect that.
```

Comment:
At least I'm pretty sure that's how it is, I haven't directly confirmed it (in the last 5 years at least)